x-pack/plugins/global_search_bar/public/plugin.tsx

```diff
@@ -52,6 +54,7 @@ export class GlobalSearchBarPlugin implements Plugin<{}, {}, {}, GlobalSearchBar
               basePathUrl={http.basePath.prepend('/plugins/globalSearchBar/assets/')}
               chromeStyle$={core.chrome.getChromeStyle$()}
               reportEvent={reportEvent}
+              maxBytes={fileUpload.getMaxBytes()}
```

Comment:
I'm loathe to just create another prop here, but I'd avoid a larger for the refactor.